doc/level1.rst

```diff
@@ -6,22 +6,51 @@
 Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
-Autocrypt makes it easy for people to encrypt email.  This document
-describes the basic capabilities required for a mail app to be
-Autocrypt-capable at Level 1.
-
-The design of Level 1 is driven by usability concerns and by the
-realities of incremental deployment. A user may mix both
-Autocrypt-enabled MUAs and traditional MUAs, and we'd
-like to avoid annoyances like unexpectedly unreadable mails while also
-supporting users who want to explicitly turn on encryption.
-
-For ease of implementation and deployment, Level 1 focuses on the use
-of Autocrypt on a single device.  We intend to :doc:`support
-multi-device synchronization (and other features) as part of Level
-2<next-steps>`.  We want to keep Level 1 simple enough that it's easy
-for developers to adopt it so we can drive efforts from real-life
-experiences as soon as possible.
+This specification is the result of maybe 2000 hours of talking, testing and
+coding iterations during 2017. Around 20 developers and many more users and
+trainers participated overall in Autocrypt sessions. The majority of these
+participants have engaged with the e-mail encryption space for a
+long time. There was overwhelming agreement that fresh approaches are needed
+and particularly ones that address usability concerns.
```

Comment:
Leading with this paragraph feels super defensive to me.  I don't think we need to be defensive.

The first paragraph ought to be "what is this thing"

If we really need a defensive paragraph, it should come at the end of this section.

Reply:
i completely stroke it for now -- i don't see it as defensive, rather as providing context of how the spec evolved to be what it is. Otherwise "autocrypt team" remains rather abstract.